dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -6,12 +6,16 @@
 - country_code = location&.country_code.to_s.upcase
 -# 'RD' is the 'Reserved' country code for private IP's, i.e. localhost.
 - us_ip = ['US', 'RD'].include?(country_code)
-- @page_title = t('activerecord.attributes.user.finish_sign_up_header')
+- if @user.errors.any?
+  - @page_title = t('activerecord.attributes.user.finish_sign_up_header_error')
+- else
+  - @page_title = t('activerecord.attributes.user.finish_sign_up_header')
```

Comment:
I would recommend moving this to a helper method like `UsersHelper.registation_page_title`
https://www.rubyguides.com/2020/01/rails-helpers/

Reply:
Why do you recommend this? The docs you linked mentioned it is helpful for shared code, but this logic is only on used once.

Reply:
It's just a bad practice to keep business logic in views: https://patrickkarsh.medium.com/importance-of-avoiding-logic-in-views-ruby-on-rails-basics-6b53f976d2f1

Reply:
It's not a blocker, just a recommendation